goald/quality/pragmatic/model/delegation.py

```diff
@@ -1,13 +1,13 @@
 from goald.quality.pragmatic.model.refinement import Refinement
 
+
 class Delegation(Refinement):
 
     def __init__(self):
         Refinement.__init__(self)
 
     def myType(self):
-        refinement = Refinement()
-        return refinement.DELEGATION
+        return Refinement().DELEGATION
 
-    def isAchivable(currentContext, interpretation):
-        return null
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)